### PR TITLE
Fixes important .md files.

### PR DESCRIPTION
**Pull request by INNOVATIVEGAMER**
**Problems:**
1.GDevelop/ExtLibs/Readme.md => (The documentation) link is not working.

**Fixed:**
Basic fixes => Current coding styles of readme like symbol usage for bold text and symbols for list creation or recommendations by prettier.

1.Core\README.md - Basic fixes.

2.GDCpp\README.md - Basic fixes.

3.GDJS\docs\Readme.md - 

- Update documentation urls to docs.gdevelop-app.com/...
- Fixed url of doxygen.org

4.GDJS\README.md - Basic and grammar fixes.

5.newIDE\README.md - Basic and grammar fixes.

6.Readme.md - Basic and grammar fixes.

7.scripts\Readme.md - Basic fixes.

8.Extensions\README.md - Basic and grammar fixes.

9.GDJS\tests\README.md - Basic and grammar fixes.

10.GDevelop.js\Readme.md - 

- Grammar fixes.
- Fixed url of nodejs.org

11.newIDE\README-extensions.md - 

- Basic and grammar fixes
- Fixes some spelling mistakes.
- Fixed url of typescriptlang.org
- Some files mentioned in the readme changed from .js to .ts as some links are not working after typescript update in GD.

12.newIDE\app\package.json.README.md - Basic and grammar fixes.

13..github\ISSUE_TEMPLATE\--feature-request.md - Basic and grammar fixes.

14.Core\GDevelop-Architecture-Overview.md -

- Basic and grammar fixes.
- Update documentation urls to docs.gdevelop-app.com/...

15.newIDE\app\public\external\jfxr\README.md - Grammar fixes.

16.newIDE\app\public\external\yarn\README.md - Grammar fixes.

17.newIDE\app\public\external\yarn\README.md - Grammar fixes.

### CODE REVIEW

**Comment by INNOVATIVEGAMER** [comment]
Please tell me the right link to "the documentation" which is currently wrong in this file : 
https://github.com/4ian/GDevelop/blob/master/ExtLibs/Readme.md

The above comment mentions the changes in each .md files. Some are just small fixes whereas some files contain some important fixes like wrong links.
@4ian I think you are busy these days , So you can review this PR when you are free ;).

File: newIDE/app/package.json.README.md [hunk 1]
```diff
@@ -1,24 +1,26 @@
...
 
 > You can check if there is only one version of a package by doing `npm ls` or `yarn why`:
-> * `yarn why react-dnd`
-> * `npm ls webpack`
+>
+> - `yarn why react-dnd`
+> - `npm ls webpack`
 
-* Latest versions of `react-sortable-hoc` seems to be breaking the lists. The exact version in which this occurs was not determined.
+- The latest versions of `react-sortable-hoc` seem to be breaking the lists. The exact version in which this occurs was not determined.
```
Comment (4ian) [inline newIDE/app/package.json.README.md:20]: Grammar mistake here

File: scripts/Readme.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Scripts files for GDevelop
 
-* **ReleaseProcedure.bat**: this script compiles, generate documentation (see **GenerateAllDocs.bat**) and package GDevelop for Windows in an installer and an archive.
-* **ReleaseProcedure.sh**: compiles and package GD for Ubuntu (see *Binaries/Packaging*).
-* **CopyWindowsToLinuxReleaseFiles.sh**: Copy all files in *Binaries/Output/Release_Windows* to *Binaries/Output/Release_Linux*. Call it after any change in *Binaries/Output/Release_Windows*.
-* **GenerateAllDocs.[bat|sh]**: Call doxygen and yuidoc to generate all the documentations into *docs* folder.
-* **ExtractTranslations.[bat|sh]**: Create the *source.pot* file containing the strings to be translated using [Crowdin](https://crowdin.com/project/gdevelop).
+-   **ReleaseProcedure.bat**: this script compiles, generate documentation (see **GenerateAllDocs.bat**) and package GDevelop for Windows in an installer and an archive.
```
Comment (4ian) [inline scripts/Readme.md:3]: Do yo know why there are some spaces after the - in some files (like here: `-   `) and not in others?

File: newIDE/README.md [hunk 3]
```diff
@@ -105,10 +105,10 @@ Any text editor is fine, but it's a good idea to have one with _Prettier_ (code
 
 ### Testing and developing with the cloud storage providers (Google Drive, Dropbox, OneDrive, etc...)
 
-Cloud storage providers are set up with development keys when you're running GDevelop in development mode. For these, to work, you must execute the web-app not from the traditional `http://localhost:3000` origin, but from `http://gdevelop-app-local:3000`:
+Cloud storage providers are set up with development keys when you're running GDevelop in development mode. For these, to work, you must execute the web-app not from the traditional `http://localhost:3000` origin, but not from `http://gdevelop-app-local:3000`:
```
Comment (4ian) [inline newIDE/README.md:108]: This fix is wrong

**Review by 4ian — APPROVED** [review]
Looks good overall, thanks! :)

File: scripts/Readme.md (hunk 2 above)
Comment (INNOVATIVEGAMER) [reply scripts/Readme.md:3]: This is just the prettier formatting. There is no impact on the preview.

File: newIDE/README.md (hunk 3 above)
Comment (INNOVATIVEGAMER) [reply newIDE/README.md]: Corrected in the latest commit.

File: newIDE/app/package.json.README.md (hunk 1 above)
Comment (INNOVATIVEGAMER) [reply newIDE/app/package.json.README.md]: Fixed in the latest commit.

**Comment by INNOVATIVEGAMER** [comment]
Applied your mentioned changes in the latest commit :)
Just out of curiosity I am asking this, Are you well these days? because there is very less activity going in the repo these days.
Or GD will get a major update ? 😕

**Comment by 4ian** [comment]
Thanks, merged!

Yes I'm all good. I'm not always directly working on code in the repository, there are also a lot of code reviews to do, and also lots of stuff to sustain the community (extensions, showcase update, website update). You can help me by ensuring that your Pull Request are as perfect as possible, as concise and clean as possible, and as small as possible. A small, well made PR will be easier to review and merge than a big one with lots of stuff to fix :)

**Pull request merged**